back/package.json

```diff
@@ -33,6 +33,7 @@
     "nexus": "^1.0.0",
     "nexus-plugin-prisma": "^0.34.1",
     "node-fetch": "^2.6.1",
+    "prisma": "^2.21.2",
```

Comment:
Prisma is already defined as a dev dependency. You can remove it from the dependencies.